selfservice/hook/session_issuer_test.go

```diff
@@ -63,8 +67,14 @@ func TestSessionIssuer(t *testing.T) {
 			f := &registration.Flow{Type: flow.TypeAPI}
 
 			require.NoError(t, reg.PrivilegedIdentityPool().CreateIdentity(context.Background(), i))
+
 			err := h.ExecutePostRegistrationPostPersistHook(w, &http.Request{Header: http.Header{"Accept": {"application/json"}}}, f, s)
-			require.True(t, errors.Is(err, registration.ErrHookAbortFlow), "%+v", err)
+			require.ErrorIs(t, err, registration.ErrHookAbortFlow, "%+v", err)
+			require.Len(t, f.ContinueWithItems, 1)
+
+			st := f.ContinueWithItems[0]
+			assert.IsType(t, &flow.ContinueWithSetToken{}, st)
```

Comment:
```suggestion
			require.IsType(t, &flow.ContinueWithSetToken{}, st)
```
because then the next assertion cannot panic, but this is more nit-picking.

Reply:
Good point, done!